tools/test/examples/examples_lib.py

```diff
@@ -398,7 +398,7 @@ def compile_repos(config, toolchains, targets, profile, verbose, examples):
                                                                 valid_choices(example['toolchains'], toolchains),
                                                                 example['features']):
                     
-                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target] + (['-vv'] if verbose else [])
+                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target, "-j", str(jobs)] + (['-vv'] if verbose else [])
```

Comment:
This forces mbed compile to be passed a jobs argument (if jobs=0 then -j0 will be specified). ; is this what you wanted?

Reply:
Yes, because 0 is default value anyway https://github.com/ARMmbed/mbed-os/blob/00f461eee01880ee185f634c6203ac7b523a7004/tools/build.py#L112

Reply:
or https://github.com/ARMmbed/mbed-os/blob/00f461eee01880ee185f634c6203ac7b523a7004/tools/make.py#L135

Reply:
Ok